### ISSUE TITLE
Attention strategy, same file to multiple sizes produces different results

### ISSUE THREAD

**Issue by ThunderRoid** [lovell/sharp#1124]
So I am using [lovell's](https://github.com/lovell/sharp/issues/792#issuecomment-298700827) method to resize a image multiple times

Code looks like this
```javascript
function resizeImage(file, dest) {
    return Promise.all([1080, 720, 480, 240, 160].map(size => {
        return sharp(file)
            .resize(size, size)
            .crop(sharp.strategy.attention)
            .toFormat(sharp.format.webp)
            .toFile(`${dest}-${size}.webp`);
    }))
}
```
Now for a sample image the **attention strategy** results in **different results**, for this image 1080 and 720 crops the face , while for other sizes, it crops in the middle.

Below is an screen shot of the results

![image](https://user-images.githubusercontent.com/3328263/36125881-cb314136-100a-11e8-867e-3c03b0acb4a7.png)

**Comment by lovell** [comment]
Hello, there's a discussion relating to this at https://github.com/jcupitt/libvips/issues/879

**Comment by zoellner** [comment]
Above link is dead. Anyone seeing this, the issue is now at https://github.com/libvips/libvips/issues/879
That issue has been closed. An improvement was shipped in libvips 8.8 on May 21

@lovell is there a way to force sharp to download v8.8.0? I'm running this in an AWS Lambda function, so global install isn't really an option

**Comment by lovell** [comment]
@zoellner If you're relying on the prebuilt binaries then you'll have to wait until sharp v0.23.0 which will provide libvips v8.8.x.
